sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_28_days_later_v1/query.sql

```diff
@@ -0,0 +1,75 @@
+WITH clients_first_seen_28_days_ago AS (
+  SELECT
+    client_id,
+    sample_id,
+    country AS country_code,
+    normalized_channel AS channel,
+    app_build_id AS build_id,
+    normalized_os AS os,
+    mozfun.norm.truncate_version(normalized_os_version, "minor") AS os_version,
+    attribution_source,
+    distribution_id,
+    attribution_ua,
+    first_seen_date AS date,  -- TODO: Why rename this?
+  FROM
+    telemetry_derived.clients_first_seen_v2
+  WHERE
+    first_seen_date = DATE_SUB(@submission_date, INTERVAL 27 day)
+    -- TODO we will need to wait till startup_profile_selection_reason is backfilled to clients_daily before uncommenting the following line
+    -- AND startup_profile_selection_reason = 'firstrun-created-default'
+),
+clients_first_seen_28_days_ago_with_days_seen AS (
+  SELECT
+    clients_first_seen_28_days_ago.*,
+    cls.days_seen_bits,
+    cls.days_visited_1_uri_bits,
+    cls.days_interacted_bits,
+  FROM
+    clients_first_seen_28_days_ago
+  LEFT JOIN
+    telemetry.clients_last_seen cls
+  ON
+    clients_first_seen_28_days_ago.client_id = cls.client_id
+    AND cls.submission_date = @submission_date
```

Comment:
Changed from post join filter to avoid ignoring clients from `clients_first_seen` that never show up in `clients_last_seen` (I don't suppose this is many but IIUC the goal is to have all clients from `clients_first_seen`)